poetry.lock

```diff
@@ -0,0 +1,975 @@
+# This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
```

Comment:
From my understanding, lockfile shouldn't be pushed on the git tree for libraries.
Am I wrong?

Reply:
> For your library, you may commit the poetry.lock file if you want to. This can help your team to always test against the same dependency versions. However, this lock file will not have any effect on other projects that depend on it. It only has an effect on the main project.
> If you do not want to commit the lock file and you are using git, add it to the .gitignore.
So I don't have experience and don't know which way is better.
Poetry install will generate poetry.lock locally anyway, maybe in case of some errors it would be easier to debug. But as we have only dev deps it shouldn't be critical.
So up to you, we can try without poetry.lock

Reply:
>  No file in /home/runner/work/aiorwlock/aiorwlock matched to [**/poetry.lock], make sure you have checked out the target repository

So I guess poetry.lock is needed for cache to work properly with setup-python

Reply:
Ok. I've reread docs. 
The lock file could be pushed into GitHub for libraries but it is not mandatory. 
Let's keep it.